worlds/stardew_valley/mods/logic/buildings.py

```diff
@@ -1,14 +1,13 @@
+from typing import Union
 from ...strings.building_names import ModBuilding
 from ..mod_data import ModNames
 from ...strings.metal_names import MetalBar
-from ...options import StardewOptions
-from ... import options
 
 
-def modded_buildings(self, world_options: StardewOptions):
+def get_modded_building_rules(vanilla_logic, active_mods: Union[bool, int, str]):
```

Comment:
Union[bool, int, str]? What the heck? isn't it neither of these 3? I thought it was a List[str] or a Set[str]?